Makefile

```diff
@@ -484,6 +484,9 @@ run-tests: all-tests
 test-results: all-tests
 	INCLUDE_SLOW_TESTS=true BINDIR=$(BINDIR) ./test.sh 2>&1 | tee ./test-results
 
+run-examples: all-examples
```

Comment:
Suggestion: If you are introducing this new target, please also include it in `help` output:

```
sdb-fdb-build:[39] $ make help
Usage: make [<target>]
Supported targets: clean all libs all-tests run-tests test-results install
```

Will be good to see `... run-tests run-examples ...` in above output.